docs/README.md

```diff
@@ -1,8 +1,14 @@
-# Documentation for Setec
+# setec
+
+Setec is a lightweight secrets management service that uses Tailscale for access control.
+
+> [!IMPORTANT]
+> File issues in our primary open source repository, [tailscale/tailscale](https://github.com/tailscale/tailscale/issues).
```

Comment:
Optional nit: I'd either leave the link bare, or maybe
```suggestion
> File issues in our [primary open source repository](https://github.com/tailscale/tailscale/issues).
```

since the link already contains the name of the repository anyway. (That said, I'm not deeply committed to this viewpoint).